compilers/concrete-compiler/compiler/lib/Dialect/FHE/Analysis/ConcreteOptimizer.cpp

```diff
@@ -304,6 +311,9 @@ struct FunctionToDag {
     return index[val];
   }
 
+  rust::Box<concrete_optimizer::Location> loc_to_location(mlir::Location location) {
+      return location_from_string(loc_to_string(location));
```

Comment:
mlir::Location can't provide the 3 optional values file, line, column ?
instead of serialize+parse

Reply:
Nope unfortunately :/